modules/virt-creating-vm-cli.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-cli_{context}"]
+= Using the CLI to create a virtual machine
+
+You can create a virtual machine from a `virtualMachine` manifest.
+
+.Procedure
+
+. Edit the `VirtualMachine` manifest, according to the following example for a RHEL virtual machine, and save it as `vm.yaml`:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    app: <vm-name>
+  name: <vm-name>
+spec:
+  dataVolumeTemplates:
+  - apiVersion: cdi.kubevirt.io/v1beta1
+    kind: DataVolume
+    metadata:
+      name: <vm-name>
+    spec:
+      sourceRef:
+        kind: DataSource
+        name: rhel9
+        namespace: openshift-virtualization-os-images
+      storage:
+        resources:
+          requests:
+            storage: 30Gi
+  running: false
+  template:
+    metadata:
+      labels:
+        kubevirt.io/domain: <vm-name>
+    spec:
+      domain:
+        cpu:
+          cores: 1
+          sockets: 2
+          threads: 1
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: rootdisk
+          - disk:
+              bus: virtio
+            name: cloudinitdisk
+          interfaces:
+          - masquerade: {}
+            name: default
+          rng: {}
+        features:
+          smm:
+            enabled: true
+        firmware:
+          bootloader:
+            efi: {}
+        resources:
+          requests:
+            memory: 8Gi
+      evictionStrategy: LiveMigrate
+      networks:
+      - name: default
+        pod: {}
+      volumes:
+      - dataVolume:
+          name: <vm-name>
+        name: rootdisk
+      - cloudInitNoCloud:
+          userData: |-
+            #cloud-config
+            user: cloud-user
+            password: '<password>'
+            chpasswd: { expire: False }
+        name: cloudinitdisk
+----
+
+. Create the virtual machine:
++
+[source,terminal]
+----
+$ oc create -f vm.yaml
+----
+
+. Start the virtual machine:
++
+[source,terminal]
+----
+$ virtctl start <vm-name>
```

Comment:
I understood that we were to use only hyphens in variable names because Kubernetes names cannot contain underscores. OTOH, I see that the OCP guidelines have underscores. So this is news to me.